### PR TITLE
Preparation of v5.3.0-CORESO-TEST

### PR DESCRIPTION
**Pull request by vbochetRTE**
* Use an environment variable to define the polling-delay for all data-bridges of a given process.
* Update itools configuration for Coreso-Test environment.
* Increase healthcheck delay from 1min to 10min for all data-bridges of all processes in all environments.
* Upgrade versions of Docker images for Coreso-Test environment.

### CODE REVIEW

**Review by TheoPascoli — APPROVED** [review]

File: k8s/overlays/coreso/test/swe-gridcapa-runner-itools-coreso-configmap.yaml [hunk 1]
```diff
@@ -47,13 +42,13 @@ data:
...
     rao-topological-actions-optimization:
-      max-search-tree-depth: 2
+      max-search-tree-depth: 3
       relative-minimum-impact-threshold: 0.0
       absolute-minimum-impact-threshold: 1.0
 
     rao-second-preventive-rao:
       execution-condition: POSSIBLE_CURATIVE_IMPROVEMENT
-      re-optimize-curative-range-actions: true
+      re-optimize-curative-range-actions: false
```
Comment (WalAmeni) [inline k8s/overlays/coreso/test/swe-gridcapa-runner-itools-coreso-configmap.yaml:51]: I don't know why this parameter was false in Azure test (I think it's an error). We should keep it true for Coreso and change it in the itools swe azure

File: k8s/overlays/coreso/test/env.properties [hunk 2]
```diff
@@ -4,62 +4,71 @@ GRIDCAPA_SOURCE_HOST_CSE_IMPORT_D2CC=192.168.236.230
 GRIDCAPA_SOURCE_PORT_CSE_IMPORT_D2CC=21
 GRIDCAPA_SOURCE_BASE_DIRECTORY_CSE_IMPORT_D2CC=/cse/import/d2cc
 GRIDCAPA_SINK_DIRECTORY_CSE_IMPORT_D2CC=/cse/import/d2cc/outputs
+GRIDCAPA_POLLING_DELAY_CSE_IMPORT_D2CC=180000
```
Comment (WalAmeni) [inline k8s/overlays/coreso/test/env.properties:7]: It's better to add these parameters also for Coreso prod for the next release

Comment (vbochetRTE) [reply k8s/overlays/coreso/test/env.properties:7]: Do you mean only add the new values in the env.properties file for prod, or also add environment variable in deployments?

Comment (WalAmeni) [reply k8s/overlays/coreso/test/env.properties:7]: Both of course ! You can keep the current value 60 s for all processes like it's in the current version installed

**Review by WalAmeni — DISMISSED** [review]

**Review by WalAmeni — APPROVED** [review]

**Pull request merged**